engine/apps/auth_token/auth.py

```diff
@@ -96,6 +96,10 @@ def _get_user(request: Request, organization: Organization) -> User:
             logger.debug(f"Could not get user from grafana request. Context {context}")
             raise exceptions.AuthenticationFailed("Non-existent or anonymous user.")
 
+    @staticmethod
+    def get_user(cls, request: Request, organization: Organization) -> User:
```

Comment:
@staticmethod doesn't receive cls argument, probably you want to use @classmethod

Reply:
exactly, thanks!